src/Uno.UWP/ApplicationModel/Contacts/ContactEmail.cs

```diff
@@ -0,0 +1,17 @@
+﻿
+namespace Windows.ApplicationModel.Contacts
+{
+	public partial class ContactEmail
+	{
+		public ContactEmailKind Kind { get; set; }
+
+		public string Address { get; set; } // 321 chars
+
+		// public string Description { get; set; } // 512 chars
+		public ContactEmail()

```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove this redundant constructor.](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=4900834)

Reply:
What is better? Adding empty constructor to implementation, or adding lines to diffignore?

Reply:
Same as previous.